src/jit/codegenarm64.cpp

```diff
@@ -416,13 +416,17 @@ void CodeGen::genSaveCalleeSavedRegistersHelp(regMaskTP   regsToSaveMask,
                                               int         lowestCalleeSavedOffset,
                                               int         spDelta)
 {
+    assert(spDelta <= 0);
     unsigned regsToSaveCount = genCountBits(regsToSaveMask);
     if (regsToSaveCount == 0)
     {
+        if (spDelta != 0)
+        {
+            genStackPointerAdjustment(spDelta, REG_NA, nullptr);
+        }
```

Comment:
For this particular case, yes the delta is only for varargs (no callee save registers) -- MAX_REG_ARG * REGSIZE_BYTES, which is 64 byte which I can just assert -- there is no need for range.
I'm a bit reluctant to add such strong assert in this API in case we might change the implementation. If immediate cannot be fit, the assert will occur down the road, ```genInstrWithConstant```.
Shall I just add some comments on it?